changelog/README.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 11th October 2024
```

Comment:
The Changelog is quite complicated. The most important things for the user are (1) What API Operations are affected by this change? (2) Are there any deprecations or breaking changes? As well as (3) What are the changes themselves. Let me work my way through the individual changes and then I will come back to this with a specific suggestion.

Reply:
I tried to improve the read me
the total change is adding new release strategy. The changes will not break request, but user should gradually replace `releasedDate` by `fixedDate` (`releasedDate` is not removed)